scripts/bootstrap.sh

```diff
@@ -111,6 +108,10 @@ fi
 if $installing; then echo "$(date -I) -- installed $version" >> /root/events.log; fi
 if $upgrading; then echo "$(date -I) -- upgraded to $version" >> /root/events.log; fi
 
+if [ "$master_ipaddr" = "masterless" ]; then
+    # ignore given IP parameter and use the one we can detect
+    master_ipaddr=$(ifconfig eth0 | awk '/inet / { print $2 }' | sed 's/addr://')
```

Comment:
turns out vagrant and masterless don't even use that value, it gets replaced almost immediately with a different minion file. I've made the fourth parameter (master ipaddr) optional and removed this section and some wrangling in bootstrap.py